DRODLib/Character.cpp

```diff
@@ -5324,17 +5324,113 @@ void CCharacter::Disappear()
 }
 
 //*****************************************************************************
-int CCharacter::GetIndexOfCommandWithLabel(const UINT label) const
+int CCharacter::GetIndexOfCommandWithLabel(const int label) const
```

Comment:
Might be better to pass the index of current label rather than loading it from current state, but I am not 100% sure.